homework4/task1.js

```diff
@@ -0,0 +1,18 @@
+const matrixArr = [
+  [1, 2, 3, 4], // → → → ↴
+  [12, 13, 14, 5], // ↱ → ↴   ↓
+  [11, 16, 15, 6], // ↑   ↑  ↲   ↓
+  [10, 9, 8, 7], // ↑ ←  ← ↲
+];
+
+function unravelArr(arr) {
+  return [
```

Comment:
u can return arr.flat().sort(...) it is easier way

Reply:
That's true thanks